package.json

```diff
@@ -2,14 +2,13 @@
   "name": "evm-mina-state",
   "version": "1.0.0",
   "dependencies": {
+    "@openzeppelin/contracts": ">=4.7.2",
     "@truffle/hdwallet-provider": "^2.0.1",
     "as-table": "1.0.55",
     "bip39": "^3.0.4",
     "eth-gas-reporter": "^0.2.24",
     "ethereumjs-wallet": "^1.0.2",
-    "web3": "^1.7.0",
-    "@nilfoundation/evm-placeholder-verification": "^1.0.10",
```

Comment:
Why should we remove this?

Reply:
Thanks for the spot. Created a task to clean this section up. Looks like an incorrect mix of deps/devDeps.